plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TemplateExample } from '@backstage/plugin-scaffolder-node';
+import yaml from 'yaml';
+
+export const examples: TemplateExample[] = [
+  {
+    description:
+      'Initializes a git repository of contents in workspace and publish it to Bitbucket Cloud with default configuration.',
+    example: yaml.stringify({
+      steps: [
+        {
+          id: 'publish',
+          action: 'publish:bitbucketCloud',
+          name: 'Publish to Bitbucket Cloud',
+          input: {
+            repoUrl:
+              'bitbucket.org?repo=repo&workspace=workspace&project=project',
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Add a description.',
```

Comment:
Should this say "Initializes a git repository in Bitbucket cloud and sets a description."

Reply:
Happy to make that change. Was following a similar convention to the GitHub action examples but agree your naming makes more sense.

Reply:
Yeah, I know what you mean. This example is definitely better than having no examples. So i would approve in any case. :)